src/test/java/ch/wisv/areafiftylan/UserRestIntegrationTest.java

```diff
@@ -1,17 +1,91 @@
 package ch.wisv.areafiftylan;
```

Comment:
It's a test class, testing all aspect of Users. What would we gain by splitting it up except for a couple of seconds when scrolling through? I don't think it's worth the extra files and lines in your debugger when running them.